src/ServiceControl.Audit/Program.cs

```diff
@@ -73,6 +77,33 @@ static Assembly TryLoadAssembly(AssemblyLoadContext loadContext, string folderPa
             return null;
         }
 
+        static void ReadExeConfiguration()
```

Comment:
I ended up moving it to ServiceControl.Configuration, which is actually shared by all three instances, and makes sense since this is related to configuration anyway.